app/views/comments/_comment.html.erb

```diff
@@ -10,12 +10,14 @@
       <p><%= comment.body %></p>
     </div>
     <div>
-      <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
-        <i class="fas fa-edit fa-fw"></i>
-      <% end %>
+      <% if current_user == comment.author %>
```

Comment:
Ah! I'm not sure how I missed this in my code, but you did good not letting it slip by! Great work!